### PR TITLE
Psr 1513

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/controllers/nonsipp/receivetransfer/RemoveTransferInController.scala
Comment (ccatosdevelopment) [inline app/controllers/nonsipp/receivetransfer/RemoveTransferInController.scala:97]: Since ReceiveTransferProgress is also included in the transferInPages list, this line can be deleted - I think this was my mistake when we were pairing, as I didn't realise this was already being removed elsewhere.

File: app/controllers/nonsipp/receivetransfer/TransferReceivedMemberListController.scala
Comment (narendrapaduchuri) [reply app/controllers/nonsipp/receivetransfer/TransferReceivedMemberListController.scala]: Hi @ccatosdevelopment ,  just pushed the latest, thanks for spotting this

**Review by ccatosdevelopment — APPROVED** [review]

**Review by kberec — APPROVED** [review]
Approving it after a rebase as it was previously approved

**Pull request merged**
